python/PiFinder/ui/object_list.py

```diff
@@ -477,11 +506,22 @@ def key_square(self):
             self.current_mode = next(self.mode_cycle)
             self.refresh()
 
-    def key_plus(self):
+    def marking_menu_left(self):
```

Comment:
maybe one method to implement with the option and the level as params?

Reply:
I was debating allowing arbitrary callable here as the API, rather than distinct callbacks.  I think with the switch to multi-level, it will need to be done with callable passed as part of the marking_menu configuration 👍